bitnami/harbor/templates/registry/registry-secret.yaml

```
@@ -48,7 +48,12 @@ metadata:
 type: Opaque
 data:
   REGISTRY_HTPASSWD: {{ $existingEnvVarsSecretRegistry.REGISTRY_HTPASSWD | default .Values.registry.credentials.htpasswd | b64enc | quote }}
-  REGISTRY_HTTP_SECRET: {{ $existingEnvVarsSecretRegistry.REGISTRY_HTTP_SECRET | default (include "common.secrets.passwords.manage" (dict "secret" (include "harbor.registry" .) "key" "REGISTRY_HTTP_SECRET" "length" 16 "providedValues" (list "registry.secret") "context" $)) }}
+  {{- $registryHttpSecret := $existingEnvVarsSecretRegistry.REGISTRY_HTTP_SECRET }}
+  {{- if eq $registryHttpSecret "" }}
+  REGISTRY_HTTP_SECRET: {{ include "common.secrets.passwords.manage" ( dict "secret" (include "harbor.registry" .) "key" "REGISTRY_HTTP_SECRET" "length" 16 "providedValues" (list "registry.secret") "context" $ ) }}
+  {{- else }}
+  REGISTRY_HTTP_SECRET: {{ $registryHttpSecret | b64enc | quote }}
```

Change to fix the linter error

```suggestion
  REGISTRY_HTTP_SECRET: {{ print $registryHttpSecret | b64enc | quote }}
```